### ISSUE TITLE
UTF8 support in kippo. 

### ISSUE THREAD

**Issue by sjums** [desaster/kippo#183]
I had someone come by and enter "weird" chars in the honeypot. This had some consequences in kippo-graph, which made me check the log. Looks like the honeypot isn't happy with utf8.

Here's my kippo-log for the incidents. https://zero.sjums.dk/?b3aa33a7fb11f4a3#AaB0/7SxECxDUAHzeF5p8zUP4ldhFuwRdhgf0SVxlEU=

**Comment by micheloosterhof** [comment]
Actually this is not a UTF-8 problem. It's the way the 'scp' file transfer protocol works.

**Comment by fgeek** [comment]
I don't think Log observer should give exceptions in that kind of situation or did I misunderstand something?

**Comment by micheloosterhof** [comment]
True, it shouldn't give exception. But 'scp' basically sends binary data to kippo, and the log observer can't handle that and chokes. 

Also since the log observer is the jsonlog component, it's also reported in the wrong repository. It must be for my version at http://www.github.com/micheloosterhof/kippo
